autoextract/aio/client.py

```diff
@@ -14,6 +14,7 @@
 from autoextract.utils import chunks
 from .retry import autoextract_retry
 from .errors import ApiError
+from ..utils import user_agent
```

Comment:
could you please move it to line 14? there is already import from this module

Reply:
Sure! I didn't notice. Fixed now